diagnostic.md

```diff
@@ -5,27 +5,30 @@ Record your responses inside the fenced code blocks below each question.
 1.  Give an example of a visual hierarchy that could be modeled with components. Explain why each piece might be it's own component.
 
     ```md
-    <!-- your response here -->
+    An example of a visial heirarch that could be modelded with components is a movie review application. The initial component could be different genres that the user can choose between. The next component, after deciding on a genre, would be a list of movie titles. The next component after the user has selected a movie could be each user's review. Each piece would be its own component because each piece must be individually reachable and editable within the hierarchy, even though they all relate to the same application. Each piece might be its own component because they each represent a view state that will undergo its own specific changes.
+
     ```
 
 1.  What is the command to generate a new component called '`my-map`'?
 
     ```sh
-    # your response here
+    ember generate component my-map
     ```
 
 1.  What files are created and/or edited to produce a component, and what are their responsibilities?
 
     ```md
-    <!-- your response here -->
+    The resulting files that are created are a component file for what ever component was generated (using `my-map` from the last question, a `my-map` component file) and a `my-map` template file.
```

Comment:
Template files (`template.hbs`) are responsible for containing markup
(defining structure), binding data, and binding and dispatching actions.

Components objects (`component.js`) are responsible for defining actions,
declaring HTML attributes, and other presentation or behavior.

In general, templates are responsible for markup, while components are
responsible for presentation and behavior.

.5/1